### DISCUSSION TITLE
Beatmap listing rarely shows same song twice

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#20169] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Rarely, the beatmap listing will show two copies of the same song. Additionally, when you begin to download one the other also shows the download progress.

This has only happened for me with the beatmap of [CORE](https://osu.ppy.sh/beatmapsets/481511#osu/1271279), and it is inconsistent.

### Screenshots or videos

<img width="1128" alt="Screen Shot 2022-09-06 at 2 50 27 PM" src="https://user-images.githubusercontent.com/100527514/188748019-83a75648-5be4-4b1e-aa85-4e9aa49080f2.png">
(When hovered or clicked on, the beatmaps are exactly identical)

### Version

2022.902.1

### Logs

[database.log](https://github.com/ppy/osu/files/9501118/database.log)
[network.log](https://github.com/ppy/osu/files/9501119/network.log)
[performance.log](https://github.com/ppy/osu/files/9501120/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9501121/runtime.log)

**Comment by smoogipoo** [comment]
This is likely going to relate to pagination.

**Comment by smoogipoo** [comment, marked as answer]
Closed the original issue to transfer to discussion. Reopened as an issue https://github.com/ppy/osu/issues/20170
